bucket/tomboy-ng.json

```diff
@@ -0,0 +1,26 @@
+{
+	"version": "0.36",
+	"description": "Tomboy-ng is a desktop simple note-taking application for Windows and other platforms",
+	"homepage": "https://github.com/tomboy-notes/tomboy-ng",
+	"license": "BSD-3-Clause-Clear",
+	"architecture": {
+		"64bit": {
+			"url": "https://github.com/tomboy-notes/tomboy-ng/releases/download/v0.36c/tomboy-ng-setup-0.36b.exe",
+			"hash": "22462dd8aebd7bcad2c1d2b387748b230ddc4c19a1a773b525737f35abb69454"
+		}
+	},
+	"checkver": "github",
+	"autoupdate": {
+		"architecture": {
+			"64bit": {
+				"url": "https://github.com/tomboy-notes/tomboy-ng/releases/download/v$versionc/tomboy-ng-setup-$versionb.exe"
```

Comment:
This app has unconventional versioning system that is a bit unpredictable. They use "a", "b", and "c" to denote minor changes. This pattern will change over time and so the downloadable version patterns will also change any time.

Adding to the complexity is that the release version, for example this 0.36c, will have executables that have a totally different version pattern, 0.36b and 0.36e.

Other apps in Scoop sometimes don't follow Semver but at the least have a consistent versioning pattern. This Tomboy app seem to have no reliable way of determining which version fits the latest version.

I will request their authors to use proper Semver. If not possible, then what we can use is the api latest release endpoint: https://api.github.com/repos/tomboy-notes/tomboy-ng/releases/latest

Then using JSON path, determine the first occurrence of an exe downloadable. Again this is not the most reliable way to do it as it can have many exe files with no clear definition of which one is the true exe file. But that's our best bet to date if the authors do not submit to Semver.